packages/DateRangePicker/src/createPropsCollector.js

```diff
@@ -0,0 +1,9 @@
+export default function createPropsCollector(displayName) {
```

Comment:
Oh! This might be a hand utility to have in the `helpers` package?

Reply:
I think so, but it is a topic I want to discuss. Introducing this helpers is just my little experiment. I see 2 issues with it:
1. It doesn't work with our README.md generator.
2. Due to components created with `createPropsCollector` have no propTypes, there is no autocompletion  for consumers. I mean when someone will install `DateRangePicker` component and will try to start typing `<DateRangePicker.Input `, there will not be any suggestions for props in editor/IDE. And I think this not solvable, because editors/IDEs can inspect only statically declared components. But we generate props collector component dynamically with this helper.

Probably, I should refactor this PR to get rid of this helper and declare subcomponents statically.

Reply:
Totally agree, props declaration should be as simple as possible abstractions should be avoided if possible, duplication for this kind of scenario is allowed so we can help static analyzer to find the better information for our users.